kubernetes/classy-vision.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: classy-vision-worker-1
+  labels:
+    job-name: classy-vision
+    worker: "1"
+spec:
+  restartPolicy: "OnFailure"
+  containers:
+  - name: classy-vision
+    image: torchelastic/examples:0.1.0rc1
+    args:
+    - "s3://torchelastic-shjiaxin-1h71m-s3bucket-m1b9b9pjldqw/petctl/shjiaxin/my_job/main.py"
```

Comment:
Since the users are expected to change this can we templetize this so that the instructions on the README can provide a `sed` command? e.g. `s3://<BUCKET>/petctl/<USER>/<JOB_NAME>/main.py`

Reply:
Sure. This is a good idea. Otherwise, user has to edit file to make the change